File: Code/App/v2/Blood/Scripts/App/Models/BloodExaminationViewModel.js
function BloodExaminationViewModel(data) {
    var self = this;

    this.PatientDieseaseId = ko.observable(data.PatientDieseaseId);

    this.PatientInfo = new PatientsModel(data.PatientInfo);
    this.BloodComment = new
        BloodExaminationCommentViewModel(data.BloodComment);
    this.examinationDescriptionBlood = ko.observable();

};

File: Code/App/separateDB/Blood/ViewModels/BloodExaminationViewModel.cs
using RepositoryClasses.Models;

namespace Blood.ViewModels
{
    public class BloodExaminationViewModel
    {
        public PatientsModel PatientInfo { get; set; }
        public BloodExaminationCommentViewModel BloodComment { get; set; }
        public int PatientDieseaseId { get; set; }
    }
}